website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -163,82 +179,81 @@ The command uninstalls the release and removes all Kubernetes resources associat
-| `applicationConfig.APPSMITH_REPLY_TO`                    | `""`  |
-| `applicationConfig.APPSMITH_MAIL_SMTP_AUTH`              | `""`  |
+| [`applicationConfig.APPSMITH_GOOGLE_MAPS_API_KEY`](/getting-started/setup/instance-configuration/google-maps)         | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_ENABLED`](/getting-started/setup/instance-configuration/email/)                | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_HOST`](/getting-started/setup/instance-configuration/email/)                   | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_PORT`](/getting-started/setup/instance-configuration/email/)                   | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_USERNAME`](/getting-started/setup/instance-configuration/email/)               | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_PASSWORD`](/getting-started/setup/instance-configuration/email/)               | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_FROM`](/getting-started/setup/instance-configuration/email/)                   | `""`  |
+| [`applicationConfig.APPSMITH_REPLY_TO`](/getting-started/setup/instance-configuration/email/)                    | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_SMTP_AUTH`](/getting-started/setup/instance-configuration/email/)              | `""`  |
 | `applicationConfig.APPSMITH_MAIL_SMTP_TLS_ENABLED`       | `""`  |
-| `applicationConfig.APPSMITH_DISABLE_TELEMETRY`           | `""`  |
+| [`applicationConfig.APPSMITH_DISABLE_TELEMETRY`](https://docs.appsmith.com/product/telemetry#disable-telemetry)           | `""`  |
 | `applicationConfig.APPSMITH_RECAPTCHA_SITE_KEY`          | `""`  |
 | `applicationConfig.APPSMITH_RECAPTCHA_SECRET_KEY`        | `""`  |
 | `applicationConfig.APPSMITH_RECAPTCHA_ENABLED`           | `""`  |
-| `applicationConfig.APPSMITH_MONGODB_URI`                 | `""`  |
-| `applicationConfig.APPSMITH_REDIS_URL`                   | `""`  |
+| [`applicationConfig.APPSMITH_MONGODB_URI`](/getting-started/setup/instance-configuration/custom-mongodb-redis#custom-mongodb)                | `""`  |
+| [`applicationConfig.APPSMITH_REDIS_URL`](/getting-started/setup/instance-configuration/custom-mongodb-redis#custom-redis)                   | `""`  |
 | `applicationConfig.APPSMITH_ENCRYPTION_PASSWORD`         | `""`  |
 | `applicationConfig.APPSMITH_ENCRYPTION_SALT`             | `""`  |
-| `applicationConfig.APPSMITH_CUSTOM_DOMAIN`               | `""`  |
+| [`applicationConfig.APPSMITH_CUSTOM_DOMAIN`](/getting-started/setup/instance-configuration/custom-domain/#dockerk8s)               | `""`  |
 
-For example, to change the encryption salt configuration, you can run the following command:
+Once you have made changes to the values file, run the below command.
 
-```
-helm install \
---set applicationConfig.APPSMITH_ENCRYPTION_SALT=123 \
-  stable-appsmith/appsmith --generate-name
+```bash
+helm install -f values.yaml stable-appsmith/appsmith --generate-name
 ```
 
-## Expose Appsmith
+## Publish
```

Comment:
this heading should either be 'Expose Appsmith server' or 'Publish Appsmith server'

Reply:
I think Publish Appsmith. Why we want to use server?

Reply:
As discussed - I will add `Publish Appsmith`